src/view/com/composer/Composer.tsx

```diff
@@ -511,8 +523,9 @@ export const ComposePost = ({
 
           <ComposerPills
             isReply={!!replyTo}
-            draft={draft}
-            dispatch={dispatch}
+            post={draft}
+            thread={composerState.thread}
+            dispatch={composerDispatch}
```

Comment:
In most cases we pass the post dispatch but here it needs the broader one to set gates. I'm just calling it `dispatch` everywhere though. Might be a bit confusing but types clarify which one it needs.